roboflow/config.py

```diff
@@ -1,26 +1,58 @@
+import json
 import os
 
+# default configuration location
+conf_location = os.getenv(
+    "ROBOFLOW_CONFIG_DIR", default=os.getenv("HOME") + "/.config/roboflow/config.json"
+)
+
+# read config file for roboflow if logged in from python or CLI
+if os.path.exists(conf_location):
+    with open(conf_location) as f:
+        config = json.load(f)
+else:
+    config = {}
+
+
+# read configuration variables conditionally
+
+
+##1. check if variable is in environment
+##2. check if variable is in config file
+##3. return default value
```

Comment:
Please, use [python docstring in google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)